static/js/layout.js

```diff
@@ -50,5 +46,56 @@ document.getElementById('user-menu').innerHTML = `
   </ul>
 `
 
-document.getElementById("userMenuVhost").onchange = (e) => Auth.selectVhost(e)
-document.getElementById("signoutLink").onclick = (e) => Auth.signOut(e)
+document.getElementById('small-menu').innerHTML = `
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
No CDN! Everything has to be hosted locally.

Reply:
and stylesheets should not be injected dynamically

Reply:
is this only for the menu icon? they maybe just download that image and include, no need to inject a mega package of icons

Reply:
Sure, will include images instead until we see that we need a range of icons then we can consider using a package.